tests/helpers/ember-power-select.js

```diff
@@ -43,15 +43,15 @@ export function getMultipleTrigger(scope) {
   return $(selector);
 }
 
-export function clickTrigger(scope, options = {}) {
+export async function clickTrigger(scope, options = {}) {
```

Comment:
Is this the reason why you needed to change te tests? Then you could keep the old version?

Reply:
The reason why we needed the new tests was that the old version produced an error that made it necessary to use the new test style.